examples/src/bin/custom_editable.rs

```diff
@@ -0,0 +1,140 @@
+//! # Custom GdkPaintable
+//!
+//! This example shows how to create a custom gdk::Paintable
+//! that renders a square with a linear color gradient
+
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use gtk::{gdk, glib, graphene, gsk};
+use std::env::args;
+
+mod imp {
+    use super::*;
+
+    pub struct CustomEditable {
+        pub text: gtk::Text,
+    }
+
+    impl Default for CustomEditable {
+        fn default() -> Self {
+            Self {
+                text: gtk::Text::new(),
+            }
+        }
+    }
+
+    #[glib::object_subclass]
+    impl ObjectSubclass for CustomEditable {
+        const NAME: &'static str = "CustomEditable";
+        type Type = super::CustomEditable;
+        type ParentType = gtk::Widget;
+        type Interfaces = (gtk::Editable,);
+
+        fn class_init(klass: &mut Self::Class) {
+            let total_props = klass.install_properties();
+            println!("total props: {}", total_props);
+            klass.set_layout_manager_type::<gtk::BoxLayout>();
+            klass.set_css_name("entry");
+            klass.set_accessible_role(gtk::AccessibleRole::TextBox);
+        }
+    }
+
+    impl ObjectImpl for CustomEditable {
+        fn set_property(
```

Comment:
Can you also add a property to `CustomEditable` so we know that it also works in combination?

Reply:
I have added a property to show/hide a spinner, like in the gtk demo